### PR TITLE
main website image update

### PR DESCRIPTION
**Pull request by hongjk1029**
image update and change request a call quote to direct whatsapp till the contact form is finish develop

### CODE REVIEW

File: frontend/src/components/Header/Header.jsx [hunk 1]
```diff
@@ -49,6 +49,10 @@ const Header = ({ isLogin, setLogin}) => {
 
   const toggleMenu = () => menuRef.current.classList.toggle("menu__active");
 
+  function navigateToWhatsApp() {
+    // window.location.href = 'https://api.whatsapp.com/send?phone=60123160808';
+    window.open('https://api.whatsapp.com/send?phone=60123160808', '_blank');
```
Comment (daniellim051000) [inline frontend/src/components/Header/Header.jsx:54]: @hongjk1029 check ur another pull request for .env

**Review by daniellim051000 — APPROVED** [review]
approve

**Comment by daniellim051000** [comment]
@hongjk1029 ur code quite old. need to pull latest file

**Comment by hongjk1029** [comment]
> @hongjk1029 ur code quite old. need to pull latest file

Ok

**Review by daniellim051000 — APPROVED** [review]

**Review by daniellim051000 — APPROVED** [review]

**Pull request merged**